hawkbit-repository/hawkbit-repository-api/src/main/java/org/eclipse/hawkbit/repository/SystemManagement.java

```diff
@@ -77,6 +80,7 @@
      *            to retrieve data for
```

Comment:
Write javadoc why this methods should use with care and the `getTenantMetadata()` should be preferred.